Source/DataSources/ModelVisualizer.js

```diff
@@ -150,6 +154,43 @@ define([
     };
 
     /**
+     * Computes a bounding sphere which encloses the visualization produced for the specified entity.
+     *
+     * @param {Entity} entity The entity whose bounding sphere to compute.
+     * @param {BoundingSphere} result The bounding sphere onto which to store the result.
```

Comment:
The bounding sphere is in the fixed frame of the scene's globe.  Which for 99.99% of our users is WGS84, but technically doesn't have to be.